test/exec.spec.js

```diff
@@ -53,7 +53,9 @@ function makeCheck (n, done) {
   }
 }
 
-describe('exec', () => {
+// TODO Needs issues with https://github.com/ipfs/js-ipfsd-ctl/pull/160#issuecomment-325669206
```

Comment:
Don't understand this comment (or the linked one either really), could we write down what's needed to make these tests pass?

Reply:
Updated the comment to present a clear action item.